fast_api_als/database/db_helper.py

```diff
@@ -343,15 +368,20 @@ def update_lead_conversion(self, lead_uuid: str, oem: str, converted: int):
         item['conversion'] = converted
         item['gsisk'] = f"1#{converted}"
         res = self.table.put_item(Item=item)
+        log_db_table_operations(res, self.update_lead_conversion, self.table.put_item)
         return True, item
 
 
-def verify_response(response_code):
-    if not response_code == 200:
-        pass
-    else:
-        pass
 
+def log_db_table_operations(res, func1, func2):
+    response_code = res['ResponseMetadata']['HTTPStatusCode']
+    if response_code == 200:
+        logging.info("[DB Helper] HTTP code returned = %d for operation %s called in function %s",response_code, func1.__name__, func2.__name__)
```

Comment:
If you're only using the function's name, then why are you using the name of the function as an argument.
It misleads anyone who is reading your code, it's completely unnecessary to pass func here.